core/frontend/src/ViewManager.ts

```diff
@@ -147,6 +147,14 @@ export class ViewManager implements Iterable<ScreenViewport> {
     this._selectedView = undefined;
   }
 
+  /** @internal */
+  public hasViewport(vpToFind: Viewport) {
+    for (const vp of this._viewports)
+      if (vp === vpToFind)
+        return true;
+    return false;
```

Comment:
github autoresolved when I clicked "Commit suggestion"

Reply:
Ends up the array is of ScreenViewports, so while direct comparison in the loop I originally had works, using .includes() does not work because .includes() expects a ScreenViewport as its argument.  Any suggestions?

Reply:
fixed